File: Chapter Revise/PrintTriangleNumbers.js
let number = 7;
let starBegin = number;
let starEnds = number;

  	for(let count = 1; count <= number; count++){
  	        let numberCheck = 2;
  	        let otherNumber = count;
  		for(let counter = 1; counter <= number * 2; counter++){

  		    	if(counter >= starBegin && counter <= starEnds) {
  		    	      if (counter <=number) process.stdout.write("" +otherNumber--);
  		    	      else process.stdout.write("" + numberCheck++);
  		    	      
  		    	      }
            else  process.stdout.write(" ");
            
  	  }
      numberCheck += 1;
  	  console.log("");  	  	      
	  starBegin-= 1; starEnds += 1;
  	}
  

  
  

File: Chapter Revise/PrintTriangleNumbers.java
public class PrintTriangleNumbers{
  public static void main (String... args){
	int number = 7;
	
	int starBegin = number;
  int starEnds = number;

  	for(int count = 1; count <= number; count++){
  	        int numberCheck = 2;
  	        int otherNumber = count;
  		for(int counter = 1; counter <= number * 2; counter++){

  		    	if(counter >= starBegin && counter <= starEnds) {
  		    	      if (counter <=number) System.out.print(otherNumber--);
  		    	      else System.out.print(numberCheck++);
  		    	      
  		    	      }
            else System.out.print(" ");
            
  	  }
      numberCheck += 1;
  	  System.out.println();  	  	      
	  starBegin-= 1; starEnds += 1;
  	}
  

  
  }
}

